apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -164,11 +164,50 @@ export class Extractor {
    * @param options - IExtractor options.
    */
   public static processProjectFromConfigFile(jsonConfigFile: string, options?: IExtractorOptions): void {
-    const configObject: IExtractorConfig = JsonFile.loadAndValidate(jsonConfigFile, Extractor.jsonSchema);
+    const configObject: IExtractorConfig = this.loadConfigObject(jsonConfigFile);
     const extractor: Extractor = new Extractor(configObject, options);
     extractor.processProject();
   }
 
+  /**
+   * Loads the api extractor config file in Extractor Config object.
+   * @param jsonConfigFile - Path to api extractor json config file.
+   */
+  public static loadConfigObject(jsonConfigFile: string): IExtractorConfig {
+    // Set to keep track of config files which have been processed.
+    const pathSet: Set<string> = new Set<string>();
+    // Get absolute path of config file.
+    let currentConfigFilePath: string = path.resolve(jsonConfigFile);
```

Comment:
>jsonConfigFile [](start = 53, length = 14)

What is this expected to be relative to? `process.cwd()`? Can you explicitly include that?

Reply:
You mean include this info in the function documentation?

---
In reply to: [236475967](https://github.com/Microsoft/web-build-tools/pull/957#discussion_r236475967) [](ancestors = 236475967)

Reply:
Added in documentation

Reply:
Fixed